exporter/src/main/java/co/rsk/core/db/bootstrap/FileExporter.java

```diff
@@ -76,7 +78,7 @@ public static void main(String[] args) throws IOException {
     }
 
     private static TrieStoreImpl getTrieStore(String originDatabaseDir) {
-        KeyValueDataSource originUnitrieDataSource = RskContext.makeDataSource("unitrie", originDatabaseDir);
+        KeyValueDataSource originUnitrieDataSource = LevelDbDataSource.makeDataSource(Paths.get(originDatabaseDir, "unitrie"));
```

Comment:
Maybe out of scope, but in the future this store could be incompatible with the "garbage collector"

Reply:
@ajlopezrsk is it right that we reference LevelDb at this level of abstraction?

Reply:
Well, it could be improved having a factory method in `RskContext`, to discuss into the core team. Now, it would be acceptable

Reply:
Ok, we can do without this now.

Reply:
> Maybe out of scope, but in the future this store could be incompatible with the "garbage collector"

Could you explain more what do you mean? Maybe it can be addressed now.

Reply:
The garbage collector use more than one folder, it has a "multitrie" implementation. Nothing to do now.